config/dispatcher/dispatcher.ex

```diff
@@ -198,6 +198,11 @@ defmodule Dispatcher do
   match "/decision-activities/*path", @json_service do
     Proxy.forward conn, path, "http://cache/decision-activities/"
   end
+  ### TODO REMOVE ME
+  match "/reports/*path", @json_service do
+    Proxy.forward conn, path, "http://cache/reports/"
+  end
+  ###
```

Comment:
TODO ?

Reply:
Will remove

Reply:
removed in [6e71daf](https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/381/commits/6e71daf31b87a6278a0959ff090632f1f09305fa)